flow/project.py

```diff
@@ -653,7 +688,7 @@ class FlowGroup(object):
     """A FlowGroup represents a subset of a workflow for a project.
 
     Any :py:class:`FlowGroup` is associated with one or more instances of
-    :py:class:`BaseFlowOperation`.
+    :py:class:`BaseFlowOperation` having identical aggregations.
```

Comment:
You can change the internal docs accordingly, but we shouldn't change the external docs to include aggregation.